Quick-Start/Prerequisite.md

```diff
@@ -1,101 +1,101 @@
- - Development Environment: SQL-based extensions are written directly in SQL and can be created using any `SQL client tool` or command-line interface that can connect to PostgreSQL.
-
-## Python:
-PostgreSQL has excellent support for the Python programming language. The `plpython` extension enables you to write extension functions using Python. Python is known for its simplicity and readability.
- - Development Environment: For Python-based extensions, you need a Python interpreter and the `psycopg2` library for connecting to PostgreSQL. You can install the required packages using `pip`.
-
-## Perl:
-Perl is a powerful scripting language, and PostgreSQL provides the `plperl` extension for writing extension functions in Perl. Perl offers rich text-processing capabilities and is well-suited for tasks involving regular expressions and text manipulation.
- - Development Environment: For Perl-based extensions, you need a Perl interpreter and the `DBD::Pg` module for PostgreSQL connectivity. You can install the required modules using `cpan` or your system's package manager.
-
-## Ruby:
-PostgreSQL supports Ruby through the `plruby` extension. Ruby is a dynamic, object-oriented language known for its concise syntax and expressiveness.
- - Development Environment: Ruby-based extensions require a Ruby interpreter and the `pg` gem for PostgreSQL connectivity. You can install the necessary gems using `gem`.
-
-## Java:
-PostgreSQL provides the `pljava` extension, which allows you to write extension functions in Java. Java is a widely used language in enterprise environments, and PL/Java enables seamless integration with PostgreSQL.
- - Development Environment: Java-based extensions require a Java Development Kit (JDK) and the PostgreSQL JDBC driver. You can set up your Java development environment by installing the JDK and adding the PostgreSQL JDBC driver to your project.
-
-## JavaScript:
-PostgreSQL supports JavaScript through the `plv8` extension. It allows you to write extension functions using JavaScript, leveraging the power of JavaScript frameworks and libraries.
- - Development Environment: JavaScript-based extensions using the `plv8` extension require the V8 JavaScript engine and the necessary development libraries. The `plv8` extension provides installation instructions for the required dependencies.
-<!-- Need to add pl/js, but it is in the initial stages (ask David and Jimmy)-->
+# What is a Postgres extension and its purpose: 
+ 
+A PostgreSQL extension is a modular unit that extends the functionality of the PostgreSQL database system. It allows you to add custom features, data types, functions, operators, and other database objects to tailor PostgreSQL to specific needs and requirements.  
+
+PostgreSQL extensions provide a way to package and distribute additional functionality for PostgreSQL in a standardized and manageable manner. Here are some key points about the purpose of PostgreSQL extensions: 
+ 
```

Comment:
Not sure why you have trailing spaces on every line, but your editor should have some command to clean them up.

Reply:
It's some issue, I checked it out. I has happened on each and every pull request form #3 to #8